test/unit/CMakeLists.txt

```diff
@@ -95,3 +98,35 @@ target_compile_definitions(
 
 # Status messages for test target
 message(STATUS "Configured unit-tests target for unit tests using Catch2")
+
+# Install Catch2 libraries
+install(
+  TARGETS Catch2
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+)
```

Comment:
Should this be something that we add as a spack dependency? Or is CMake okay here since this is header only (or something like that...)

Reply:
Good point! I'll add it to spack.